### DISCUSSION TITLE
Override Item Scaling Costs

### DISCUSSION THREAD

**Discussion by VoltCruelerz** [misthero/dnd5e-spellpoints#129] [Q&A]
I have both a spell point sorcerer and a [KibblesTasty psion](https://drive.google.com/file/d/1Vswceq3NJi2_qqQXmAUH3k0bP1cagCDz/view) at my table. I'd love to be able to use this mod for both, but at present I can only use it for one or the other because the costs per spell level differ for them.

Would it be possible to get an option for an item (in practice, a spell) to override the default costs? I could see this being useful for things like Aberrant Mind sorcerer's Psionic Sorcerery too.

**Comment by misthero** [comment]
> I have both a spell point sorcerer and a [KibblesTasty psion](https://drive.google.com/file/d/1Vswceq3NJi2_qqQXmAUH3k0bP1cagCDz/view) at my table. I'd love to be able to use this mod for both, but at present I can only use it for one or the other because the costs per spell level differ for them.
> 
> Would it be possible to get an option for an item (in practice, a spell) to override the default costs? I could see this being useful for things like Aberrant Mind sorcerer's Psionic Sorcerery too.

did you tried the new version?

**Comment by misthero** [comment]
this is possible configuring 2 different spell points items.
